formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -30,28 +29,27 @@ let tmp = {
 /**
  * Comment on stackWithAttributes.
  */
```

Comment:
@jordwalke When I see it again I still like this form better? Another solution is to add an empty line after the comment:
```
class virtual stackWithAttributes 'a init => {
/* Before class */

/* The "as this" should not be formatted away because attributes. */
....
```